src/services/files.js

```diff
@@ -74,4 +76,82 @@ module.exports = class FilesHelper {
 			throw error
 		}
 	}
+
+	static async getSignedUrlByOrgId(fileName, id, orgId, dynamicPath) {
```

Comment:
Why do we need this function?

Reply:
@nevil-mathew here we are getsignedUrl based on orgId , if they need to create a custom csv for particular
orgId , i am using this function.

Reply:
What does signedURL have to do with org id?